core/templates/services/exploration-html-formatter.service.ts

```diff
@@ -65,12 +71,21 @@ export class ExplorationHtmlFormatterService {
     element = (
       this.extensionTagAssembler.formatCustomizationArgAttrs(
         element, interactionCustomizationArgs));
-    element.attr(
-      'last-answer', parentHasLastAnswerProperty ? 'lastAnswer' : 'null');
     if (labelForFocusTarget) {
       element.attr('label-for-focus-target', labelForFocusTarget);
     }
-    return element.get(0).outerHTML;
+    element.attr(
+      'last-answer', parentHasLastAnswerProperty ? 'lastAnswer' : 'null');
+    let val = element.get(0).outerHTML;
+    if (this.migratedInteractions.indexOf('GraphInput') >= 0) {
+      val = val.replace(
+        'last-answer="null"></oppia-interactive-graph-input>',
+        '[last-answer]="null"></oppia-interactive-graph-input>');
+      val = val.replace(
+        'last-answer="lastAnswer"></oppia-interactive-graph-input>',
+        '[last-answer]="lastAnswer"></oppia-interactive-graph-input>');
```

Comment:
Would this work?
```suggestion
      val = val.replace('last-answer=', '[last-answer]=');
```

Reply:
My fear is that the characters in the suggested replacement might open up XSS attacks. This has to be as specific as possible.